grails-app/migrations/views/order-summary-materialized-view.sql

```diff
@@ -0,0 +1,3 @@
+DROP TABLE IF EXISTS mv_order_summary;
+CREATE TABLE mv_order_summary AS SELECT * FROM order_summary;
```

Comment:
Careful here. There's going to be a time where this table is empty / missing and all queries will fail. We want that time to be as short as possible. So it would be better to create a temporary table mv_order_summary_temp) to load the data from the order_summary view and then replace mv_order_summary with that table i.e. either through a table rename or a similar CREATE TABLE SELECT or INSERT INTO SELECT.

Reply:
Although I guess this is fine since it's just the initial load.

Reply:
Fixed it